### PR TITLE
print things a little closer to how mathsteps currently prints

### PR DESCRIPTION
**Pull request by evykassirer**
There were two big changes when I switched to math-parser print, which I updated here.

If you think these shouldn't be part of the printing, I can change it in mathsteps (it is just cosmetic preference) but if you like it then I get to keep my cosmetic preferences and don't have to go through and change all the tests! ^_^

Let me know what you think!

### CODE REVIEW

File: lib/__test__/print.test.js [hunk 1]
```diff
@@ -36,8 +36,10 @@ describe('print', () => {
     describe('fractions', () => {
         const tests = [
             ['(x + 1) / 1', '(x + 1) / 1'],
-            ['1/2/3', '1 / 2 / 3'], // (1/2) / 3
-            ['1*2/3', '1 * 2 / 3'], // 1 * (2/3)
+            ['2/3 + 5', '2/3 + 5'],
+            ['2/x + 5', '2 / x + 5'],
```
Comment (k4b7) [inline lib/__test__/print.test.js]: It seems odd to make a distinction between `2/3` and `2/x`.

File: lib/__test__/print.test.js [hunk 2]
```diff
@@ -36,8 +36,10 @@ describe('print', () => {
...
+            ['1/2/3', '1/2 / 3'], // (1/2) / 3
+            ['1*2/3', '1 * 2/3'], // 1 * (2/3)
```
Comment (k4b7) [inline lib/__test__/print.test.js]: This is definitely an improvement.

File: lib/__test__/print.test.js (hunk 1 above)
Comment (evykassirer) [reply lib/__test__/print.test.js]: wanna do it for all fractions where numerator and denominator are both either identifier or number?

Comment (k4b7) [reply lib/__test__/print.test.js]: I'd prefer that.  Hopefully that won't cause too much troubles with existing mathsteps tests.

**Comment by k4b7** [comment]
@evykassirer thanks.  I'll do a publish this evening.

**Comment by evykassirer** [comment]
awesome! let me know once it's published and I'll go in and clean up my mathsteps PR a bit with that (I think it'll help a lot :D)

**Comment by k4b7** [comment]
I've published 0.12.0.

**Pull request merged**